torchci/pages/minihud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -242,15 +242,15 @@ function CommitSummaryLine({
       )}
       {ttsAlert && (
         <span style={{ float: "right" }}>
-          <b>tts alert</b>
+          <ShowDurationInfoCheckbox sha={row.sha} />
         </span>
       )}
       <CommitLinks row={row} />
     </div>
   );
 }
 
-function getTTSChanges(jobs: JobData[], prevRow: RowData | undefined) {
+function getTTSChanges(jobs: JobData[], prevRow: JobData[] | undefined) {
```

Comment:
Nits: may be change this variable to `prevJobs` now to reflect what it has become